scarb/tests/fetch.rs

```diff
@@ -0,0 +1,18 @@
+
+
+use scarb_test_support::command::Scarb;
+use scarb_test_support::project_builder::ProjectBuilder;
+
+#[test]
+fn simple() {
+    let t = assert_fs::TempDir::new().unwrap();
+    ProjectBuilder::start()
+        .name("hello")
+        .version("0.1.0")
+        .build(&t);
+
+    Scarb::quick_snapbox()
+        .arg("fetch")
+        .current_dir(&t);
+}
```

Comment:
Would be cool to verify if downloading dependencies has actually been triggered. 
We can do that by: 
1. Adding a simple git dependency 
```
use indoc::indoc;

use scarb_test_support::gitx;

(...)

    let git_dep = gitx::new("dep1", |t| {
        ProjectBuilder::start()
            .name("dep1")
            .build(&t)
    });

(...)
    
    ProjectBuilder::start()
        .name("hello")
        .version("0.1.0")
        .dep("dep1", &git_dep)

(...)
```

2. Verifying the command stdout for info about downloading it 

```
(...)
  .assert()
  .success()
  .stdout_matches(indoc! {r#"
  [..]  Updating git repository file://[..]/dep1
  "#});
```